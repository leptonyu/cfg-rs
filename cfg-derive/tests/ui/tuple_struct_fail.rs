use cfg_derive::FromConfig;

#[derive(FromConfig)]
struct T(u8, u8);

fn main() {}
