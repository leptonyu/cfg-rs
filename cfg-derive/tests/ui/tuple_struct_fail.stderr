error: proc-macro derive panicked
 --> tests/ui/tuple_struct_fail.rs:3:10
  |
3 | #[derive(FromConfig)]
  |          ^^^^^^^^^^
  |
  = help: message: Only support named body
